### ISSUE TITLE
Getting datetimes in UTC

### ISSUE THREAD

**Issue by FlorEtcheverry** [RIPE-NCC/ripe-atlas-cousteau#36]
Hello, I'm using the API's meta data from Measurement to identify the number of iteration (using the timestamp of a result, the start time and the interval).

When I was testing this, the differences didn't make sense, since I'm using UTC values and I assumed `msm.start_time` would return a timestamp in UTC (as seen on, for example, https://atlas.ripe.net/api/v2/measurements/4496949/), but it returns a naive datetime object in localtime. (My local time is GMT+2:00 DST). Also, I'm using Python 3.2 so I don't have `datetime.timestamp()`.

I couldn't found specification of this in the docs, so I took a look at the code. I found in [api_meta_data.py](https://github.com/RIPE-NCC/ripe-atlas-cousteau/blob/master/ripe/atlas/cousteau/api_meta_data.py) that the method `populate_times()` uses `datetime.fromtimestamp()` that returns the local date and time, instead of `datetime.utcfromtimestamp()`.
I think maybe using UTC values would be better since the timestamps in Ripe are in UTC.

Thank you.

**Comment by danielquinn** [comment]
Good point.  I agree that timestamps should always be in UTC, but it should be noted that `datetime.utcfromtimestamp()` actually returns a naïve datetime object, it just makes sure that the datetime given is a UTC one.  If we want a timezone-aware timestamp (and I would argue that we do), we should probably be using `arrow.get()` instead:

```python
>>> from datetime import datetime
>>> datetime.utcfromtimestamp(1470006000)
datetime.datetime(2016, 7, 31, 23, 0)

>>> import arrow
>>> arrow.get(1470006000).datetime
datetime.datetime(2016, 7, 31, 23, 0, tzinfo=tzutc())
```

**Comment by FlorEtcheverry** [comment]
Yes. I've also found `pytz` library that I think it does the same.
This could be taken into account when receiving times too (if it's not already).

**Comment by astrikos** [comment]
But since we have `dateutil` as a [requirement](https://github.com/RIPE-NCC/ripe-atlas-cousteau/blob/master/setup.py#L10) it makes sense to use that :)
```python
>>> from datetime import datetime
>>> from dateutil.tz import tzutc
>>> dt = datetime.utcfromtimestamp(1470006000)  # datetime.datetime(2016, 7, 31, 23, 0)
>>> newdt = dt.replace(tzinfo=tzutc())
>>> newdt
datetime.datetime(2016, 7, 31, 23, 0, tzinfo=tzutc())
```
I will try to change it next time unless someone has time and do a pull request until then :)

**Comment by danielquinn** [comment]
Ah you're right.  Sorry, I thought we already had arrow in the requirements.  Since that's just a friendly wrapper around dateutil, it makes much more sense to just use that in this case.

**Comment by astrikos** [comment]
i consider that fixed with #37
